pkg/controllers/status/workstatus_controller.go

```diff
@@ -198,7 +198,7 @@ func (c *WorkStatusController) handleDeleteEvent(key keys.FederatedKey) error {
 
 	// Given the workload might has been deleted from informer cache, so that we can't get work object by it's label,
 	// we have to get work by naming rule as work's name also is binding's name and binding's name consist of workload's name and kind.
```

Comment:
The comment needs an update as well, right?

Reply:
Right👍, I will fix it.